### PR TITLE
Remove //nolint: gas directives from fmt.Fprintf #590

### LINKED ISSUE #590: Filter out `gas` linter error checks for `fmt.Fprintf` commands
# Problem

We have a lot of unnecessary `// nolint: gas` comments near `fmt.Fprintf`.
We never want to check error value of `Fprintf` anyway.

# Implementation

1. Remove `// nolint: gas` comments from `fmt.Fprintf`
2. Tweak the `gometalinter` config to ignore these warnings from `gas`. Keep the other warnings intact.

# Acceptance

1. There are no `// nolint: gas` comments near `fmt.Fprintf`.
2. `make lint` shows no warnings
3. Other warnings from `gas` aren't ignored.

### PR DESCRIPTION
**Pull request by alexkohler**
**Summary:**
Filter out gas linter error checks for `fmt.Fprintf` commands. This required defining a custom linter around gas that additionally included the offending code.

**Notes:**

Gas format, without piping it through gometalinter, gives output like this: 
```Bash
$ gas -fmt=csv geth/jail/console/console.go
geth/jail/console/console.go,21,Errors unhandled.,LOW,HIGH,"fmt.Fprintf(w, ""%s: %s"", consoleEventName, formatForConsole(fn.ArgumentList))"
```

Gometalinter, by default, does not grab the line of code when it filters `gas` errors. To resolve this, I created a wrapper around `gas` (I wasn't sure what to call this "gas wrapper", I opted for gasv2, open to other names). 

The first part of the regular expression was taken directly from gometalinter (see https://github.com/alecthomas/gometalinter/blob/master/linters.go#L236), and I then appended `,\".*\"` to additionally grab the line of code of the offending line. Lastly, I excluded `".*Errors unhandled.*fmt.Fprintf.*"` to filter out only fmt.Fprintf errors around omitted errors.

Also as a result of this change, gas lint output will now include the offending code.

Closes #590

### CODE REVIEW

**Comment by dshulyak** [comment]
nice, the only thing that I can't understand - why not to include Fprint and Fprintln?

**Comment by dshulyak** [comment]
@alexkohler by the way, is it intentional that commits are not attached to your github account?

File: .gometalinter.json [hunk 1]
```diff
@@ -1,15 +1,22 @@
 {
     "Exclude": [
         ".*_mock.go",
-        "geth/jail/doc.go"
+        "geth/jail/doc.go",
+        ".*Errors unhandled.*fmt.Fprintf.*"
```
Comment (mandrigin) [inline .gometalinter.json]: Can we also make sure that it is `gas`? It should print out its name too.

**Review by mandrigin — CHANGES_REQUESTED** [review]
Looks good! One small question.

**Comment by alexkohler** [comment]
@dshulyak - Good call. I'll update the regex to .`*fmt.Fprint.*` so it'll grab Fprint, Fprintf, and Fprintln. 

@dshulyak  - That's not intentional, I'm not sure why that's happening... (on a side note, I apologize for the sporadic commits in this PR). Would you like me to try opening another PR?

File: .gometalinter.json (hunk 1 above)
Comment (alexkohler) [reply .gometalinter.json]: Sure.

**Comment by dshulyak** [comment]
@alexkohler sporadic commits are not a problem, you can always squash them (and just push to the same branch) or we can do it on a github when PR will be merged.

**Comment by alexkohler** [comment]
To clarify, here is what a gasv2 message looks like (this is why gasv2 is at the end of the `Exclude` filter):

`geth/jail/console/console.go:21::warning: Errors unhandled.,LOW,HIGH,"fmt.Fprintf(w, ""%s: %s"", consoleEventName, formatForConsole(fn.ArgumentList))" (gasv2)`

**Review by mandrigin — APPROVED** [review]
Looks good!

**Review by dshulyak — APPROVED** [review]

**Pull request merged**